ooc-cli.use

```diff
@@ -0,0 +1,6 @@
+Name: ooc CLI
+Description: CLI helpers
+SourcePath: source/cli
+Requires: ooc-collections
```

Comment:
What does this do? Do you still need `use ooc-collections` in `ArgumentParser.ooc`?